libs/wire-api/src/Wire/API/Team/Role.hs

```diff
@@ -106,6 +108,21 @@ instance FromJSON Role where
     -- ~fisx, Wed Jan 23 16:38:52 CET 2019
     bad -> fail $ "not a role: " <> show bad
 
+instance ToByteString Role where
```

Comment:
pls add round-trip test.

Reply:
(and pls make sure the arbitrary instance doesn't need to be changed if new constructors are added (something involving `[minBound..]`))

Reply:
also, i would implement the aeson instances in terms of the bytestring instances, now that you have the latter.  but feel free to disagree.

Reply:
Done, with separate tests for both: aeson and bytestring